README.md

```diff
@@ -77,6 +77,11 @@ Python 2, make sure the virtual environment will use Python 3:
     cd dencam
     python setup.py install
 
+### Install dencam dependencies with optional dependencies
+    
```

Comment:
Perhaps it would be worth explaining just a little about what things require the optional dependencies i.e. what tools use them.

Reply:
I definitely agree. I'll get around to doing that either in the evening or tomorrow afternoon.